src/main/java/mate/controller/AddCarController.java

```diff
@@ -0,0 +1,44 @@
+package mate.controller;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.model.Car;
+import mate.model.Manufacturer;
+import mate.service.CarService;
+import mate.service.ManufacturerService;
+
+@WebServlet(urlPatterns = "/add-car")

```

Comment:
Use good URL naming instead of bad ones. Same for all.
![image](https://user-images.githubusercontent.com/55624671/169707021-45e1a2b5-ac41-4558-81f0-2e14d3480175.png)